scripts/generate/js_api.js

```diff
@@ -201,10 +201,10 @@ module.exports = function (branch, done) {
         def.documentation = 'http://www.elasticsearch.org/guide/en/elasticsearch/reference/master/cat.html';
       }
 
-      def.documentation = def.documentation.replace(
+      def.documentation = def.documentation ? def.documentation.replace(
```

Comment:
Required as there are no docs for painless context API: https://github.com/elastic/elasticsearch/blob/7.x/rest-api-spec/src/main/resources/rest-api-spec/api/scripts_painless_context.json